easybuild/easyconfigs/r/RStudio-Server/rstudio-1.2.1335.patch

```diff
@@ -0,0 +1,11 @@
+diff -Nru rstudio-1.2.1335.orig/src/cpp/core/system/PosixSystem.cpp rstudio-1.2.1335/src/cpp/core/system/PosixSystem.cpp
```

Comment:
@ocaisa What's the source for these patches? Author or upstream reference? Also missing a description.

We usually also name them to match with the software name, so `RStudio-Server-1.2.1335_fix-foo.patch` (where `foo` is something more meaningful, of course).

Reply:
These come unchanged from from https://github.com/ComputeCanada/easybuild-easyconfigs/tree/computecanada-main/easybuild/easyconfigs/r/RStudio-Server